cirq-core/cirq/transformers/analytical_decompositions/single_to_two_qubit_isometry_test.py

```diff
@@ -61,6 +61,8 @@ def test_two_qubit_matrix_to_cz_isometry(
     decomposed_final_state_vector = sim.simulate(decomposed_circuit).final_state_vector
 
     assert (
-        cirq.allclose_up_to_global_phase(original_final_state_vector, decomposed_final_state_vector)
+        cirq.allclose_up_to_global_phase(
+            original_final_state_vector, decomposed_final_state_vector, atol=1e-5
```

Comment:
Tolerance of 1e-5 seems a little excessive. Maybe reduce to 1e-6 or 5e-7?

```console
>>> import cirq
>>> cirq.allclose_up_to_global_phase(a, b)
False
>>> cirq.allclose_up_to_global_phase(a, b, atol=1e-5)
True
>>> cirq.allclose_up_to_global_phase(a, b, atol=1e-6)
True
>>> cirq.allclose_up_to_global_phase(a, b, atol=1e-7)
True
>>> cirq.allclose_up_to_global_phase(a, b, atol=1e-8)
False
>>> a[0]
(-0.3801833+0.01406243j)
>>> b[0]
(-0.21030015+0.3170347j)
>>> f = a[0]/b[0]
>>> f
(0.5832012+0.81232756j)
>>> a
array([-0.3801833 +0.01406243j,  0.5599068 -0.11519583j,
       -0.32348925+0.04328994j, -0.3777393 +0.05938133j,
       -0.26665634+0.04652762j, -0.3081506 +0.17547803j,
       -0.00454243+0.2769741j ,  0.00223864-0.00217629j], dtype=complex64)
>>> b*f
array([-0.38018334+0.01406242j,  0.55990684-0.11519577j,
       -0.32348925+0.04328991j, -0.37773922+0.05938125j,
       -0.2666563 +0.04652753j, -0.30815056+0.17547797j,
       -0.00454246+0.27697408j,  0.0022386 -0.00217629j], dtype=complex64)
>>> a - b*f
array([ 2.9802322e-08+1.0244548e-08j, -5.9604645e-08-6.7055225e-08j,
        0.0000000e+00+2.6077032e-08j, -8.9406967e-08+8.1956387e-08j,
       -2.9802322e-08+8.5681677e-08j, -2.9802322e-08+5.9604645e-08j,
        2.5145710e-08+2.9802322e-08j,  4.4004992e-08+1.8626451e-09j],
      dtype=complex64)
>>> np.abs(a - b*f)
array([3.1513952e-08, 8.9716877e-08, 2.6077032e-08, 1.2128666e-07,
       9.0716746e-08, 6.6640020e-08, 3.8993399e-08, 4.4044398e-08],
      dtype=float32)
>>> np.max(np.abs(a - b*f))
1.2128666e-07
>>> np.linalg.norm(np.abs(a - b*f))
2.0140389e-07
```

Reply:
Changed to 1e-6